packages/web/components/token-details/token-details.tsx

```diff
@@ -0,0 +1,130 @@
+import React, { useState } from "react";
+
+import { Icon } from "~/components/assets";
+import LinkIconButton from "~/components/buttons/link-icon-button";
+
+const TEXT_CHAR_LIMIT = 485;
+
+function TokenDetails() {
+  const [isExpanded, setIsExpanded] = useState(false);
+  const text = `Osmosis is a decentralized exchange (DEX) that uses the Cosmos SDK for optimized functionality with other blockchains. Its automated market makers set it apart, allowing enhanced market responsiveness for liquidity providers.\n\nOsmosis is a multi-chain DEX specifically built to function with other blockchains in the Cosmos ecosystem. Communication withthese other blockchains is achieved via its Inter-Blockchain Communication protocol (IBC). Because Osmosis is linked to other Tendermint blockchains, providing smooth, fast trades,`;
+
+  return (
+    <div className="flex flex-col items-start gap-3 self-stretch rounded-5xl border border-osmoverse-800 bg-osmoverse-900 p-10 xl:gap-6 1.5xs:gap-6">
```

Comment:
Maybe we can reduce padding on smaller device, like we do for "Related assets" section